src/forms/FormAutocompleteTag.jsx

```diff
@@ -0,0 +1,439 @@
+
+      _setValue(newTags);
+      if (isFunction(afterChange)) {
+        afterChange(newTags, previousValue);
+      }
+    },
+    [_setValue, afterChange, value]
+  );
+
+  const addTag = useCallback(
+    (newTag = '') => {
+      if (isString(newTag) && !allowUnlistedValue) {
+        return;
+      }
+
+      const selectedTags = value?.map((tag) => {
+        if (trackBy && isObject(tag)) {
+          return getValueByPath(tag, trackBy) ?? '';
+        }
+
+        return isDefined(tag?.label) ? tag.label : tag;
+      });
+
+      if (
+        (trackBy && isObject(newTag) && selectedTags?.includes(getValueByPath(newTag, trackBy))) ||
+        selectedTags?.includes(newTag)
+      ) {
+        return showToast(repeatedTagErrorMessage, {
+          type: 'danger',
+        });
+      }
+
+      const newTags = [...value];
+
+      if (!newTag?.label) {
+        newTags?.push({
+          label: newTag,
+          value: newTag,
+        });
+      } else {
+        newTags?.push(newTag);
+      }
+
+      setValue(newTags);
+    },
+    [allowUnlistedValue, repeatedTagErrorMessage, setValue, showToast, trackBy, value]
+  );
+
+  const onSearchInputFocus = useCallback(() => {
+    if (openOnFocus) {
+      setTimeout(() => {
+        open();
+      }, 100);
+    }
+  }, [open, openOnFocus]);
+
+  const onSearchInputBlur = useCallback(() => {
+    if (isEmptyLike(searchValue) && value) {
+      updateSearchInputValidation();
+    }
+
+    if (ignoreBlur) {
+      searchInputRef.current.focus();
+    } else {
+      close();
+      setFocus(false);
+    }
+  }, [close, ignoreBlur, searchValue, updateSearchInputValidation, value]);
+
+  const enableSearchInput = useCallback(() => {
+    if (disabled) {
+      return;
+    }
+
+    setFocus(true);
+    setTimeout(() => {
+      searchInputRef.current.focus();
+    });
+  }, [disabled]);
+
+  const onSelectItem = useCallback(
+    ({ value }) => {
+      addTag(value);
+      setSearchValue('');
+      setIgnoreBlur(false);
+      setTimeout(() => {
+        setFocus(false);
+        close();
+      }, 60);
+    },
+    [close, addTag]
+  );
+
+  const onClick = useCallback(
+    (e) => {
+      e.preventDefault();
+      e.stopPropagation();
+      onSelectItem({ value: searchValue });
+    },
+    [onSelectItem, searchValue]
+  );
+
+  const onKeyDown = useCallback(
+    (e) => {
+      e.stopPropagation();
+      if (allowUnlistedValue && e.key === 'Enter') {
+        e.preventDefault();
+        addTag(searchValue);
+        setSearchValue('');
+      }
+    },
+    [addTag, searchValue, allowUnlistedValue]
+  );
+
+  useEffect(updateSearchInputValidation, [updateSearchInputValidation]);
+
+  useEffect(() => {
+    setControlFeedback(formSubmitedAttempted ? (isValid() ? 'is-valid' : 'is-invalid') : '');
+  }, [controlFeedback, formSubmitedAttempted, getFormSubmitedAttempted, isValid]);
+
+  return (
+    <div className={formatClasses(['d-flex', 'mb-3', 'form-input-tag', controlFeedback])}>
```

Comment:
Para o feedback de validação funcionar corretamente no `FormGroup`, a tag ""mais de fora"" do componente deve ter o "is-valid" ou o "is-Invalid"